fix: re-add support for inline termsOfService

marcelstoer opened the pull request:
### Motivation and Context
Addresses https://github.com/swagger-api/swagger-ui/pull/5341#pullrequestreview-562790852

### How Has This Been Tested?


### Screenshots (if appropriate):



## Checklist

### My PR contains... 
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

tim-lai commented:
@marcelstoer Thanks for the PR! It looks good, however, it could use a test to validate the behavior.

marcelstoer commented:
Sorry Tim, I really have no idea what I'm doing here 🤣 JavaScript et.al. is not my universe.

i added what I think should be tested. They pass, but I don't think they're actually executing the code I modified in `render()`.

tim-lai commented:
@marcelstoer Thanks for the update. I think an easier to implement test would to be use Cypress. Add a mock definition as you described in the issue, render it in Cypress, then check that the relative url element is as expected.
